docs/installation/docker.md

```diff
@@ -110,7 +110,3 @@ Note that uploaded files, like avatars, will disappear when you
 restart the container. Map a volume to
 `/var/www/monica/storage/app/public` if you want that data to persist
 between runs. See `docker-compose.yml` for examples.
-
-#### Other documents to read
-
-[Connecting to MySQL inside of a Docker container](https://github.com/monicahq/monica/wiki/Installation-with-Docker:-connection-to-MySQL)
```

Comment:
Well @djaiss, any idea where this doc is ?

Reply:
I can grab the copy from https://github.com/monicahq/monica/blob/40bc3373d8536ae5d05fe2c676567ff9e89ae12c/docs/database/connecting.md but IDK if it was updated while it was on the wiki.